src/WheelCurvedPicker.android.js

```diff
@@ -1,5 +1,5 @@
 import React, { PureComponent } from 'react';
-import { ColorPropType, requireNativeComponent, View } from 'react-native';
+import { ColorPropType, requireNativeComponent, View, ViewPropTypes } from 'react-native';
```

Comment:
To continue to support proptypes for RN < 0.44 ([when `ViewPropTypes` was created](https://github.com/facebook/react-native/releases/tag/v0.44.3)), you can do as follows: 

```es6
import { ..., View, ViewPropTypes as RNViewPropTypes } from 'react-native';

const ViewPropTypes = RNViewPropTypes || View.propTypes;
```

Reply:
Good point, Thank you.